cirq-google/cirq_google/calibration/workflow_test.py

```diff
@@ -1181,7 +1181,7 @@ def test_run_floquet_characterization_for_moments():
     itertools.product([0.1, 0.7], [-0.3, 0.1, 0.5], [-0.3, 0.2, 0.4], [-0.6, 0.1, 0.6], [0.2, 0.6]),
 )
 def test_fsim_phase_corrections(
-    theta: float, zeta: float, chi: float, gamma: float, phi: float
+    theta: cirq.value.TParamVal, zeta: float, chi: float, gamma: float, phi: cirq.value.TParamVal
```

Comment:
I think this is available on the main `cirq` namespace, so can just do
```suggestion
    theta: cirq.TParamVal, zeta: float, chi: float, gamma: float, phi: cirq.TParamVal
```
Can do this throughout cirq_google, since it's external to cirq_core.